jmeter-service/Dockerfile

```diff
@@ -56,7 +60,8 @@ RUN    apk update \
 	&& curl -L --silent ${JMETER_DOWNLOAD_URL} >  /tmp/dependencies/apache-jmeter-${JMETER_VERSION}.tgz  \
 	&& mkdir -p /opt  \
 	&& tar -xzf /tmp/dependencies/apache-jmeter-${JMETER_VERSION}.tgz -C /opt  \
-	&& rm -rf /tmp/dependencies
+	&& rm -rf /tmp/dependencies \
+	&& curl -L --silent ${DYNATRACE_EXTENSION_URL} > /opt/apache-jmeter-${JMETER_VERSION}/lib/ext/jmeter-dynatrace-plugin-${DYNATRACE_EXTENSION_VERSION}-SNAPSHOT.jar
```

Comment:
I think this is okay for the time being, but long-term we **most not** download additional dependencies/plugins for JMeter and put them into the container. 
We will end up getting 100 of those dependencies and we will be responsible for ensuring that all of them are up2date, and if anything breaks it will also be our job to investigate.

Reply:
I have created a follow-up issue on that: https://github.com/keptn/keptn/issues/2832

Reply:
agree - but - I was only able to add this plugin and so far I havent come across any other plugins that make sense. if anybody else wants to add plugins feel free to suggest